.github/workflows/ci.yml

```diff
@@ -100,7 +78,5 @@ jobs:
       run: |
         source .venv/bin/activate
         pip install .
-        osivalidator --data data/20210818T150542Z_sv_312_50_one_moving_object.txt -r rules
-        osivalidator --data data/20210818T150542Z_sv_312_50_one_moving_object.txt -r rules --parallel
         osivalidator --data data/20210818T150542Z_sv_312_50_one_moving_object.osi -r rules
         osivalidator --data data/20210818T150542Z_sv_312_50_one_moving_object.osi -r rules --parallel
```

Comment:
As the `--parallel` argument does not have an effect anymore, this second CI run could be removed as well.

Reply:
Sure one could, but it does not cost us any and is still a valid test, and protects us against problems if we reintroduce parallel in the next refactoring, so would leave this in for now.

Reply:
While you are perfectly right, I am always concerned that new contributors or even users might look at the tests for examples. But from my understanding the whole program arguments should be refactored later (as proposed by you in the other thread), so lets keep it like this.